### PR TITLE
ERD, data model adjustment, docs, apps

### PR DESCRIPTION
**Pull request by natsn**
Minor changes

### CODE REVIEW

File: project/project/models.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ class Profile(models.Model):
 
     TIMEZONES = [(tz,tz) for tz in pytz.all_timezones]
     timezone = models.CharField(max_length=100, choices=TIMEZONES, default='US/Pacific')
-    def is_with(self, profile):
+    def belongs_to(self, profile):
```
Comment (Borgaard) [inline project/project/models.py]: :+1:

File: project/project/models.py [hunk 2]
```diff
@@ -143,6 +150,7 @@ class Media(models.Model):
     file = models.FileField(upload_to = u'uploads/')
 
 class Tip(models.Model, TranslatedModelMixin):
+    curriculum = models.ForeignKey(Curriculum)
```
Comment (Borgaard) [inline project/project/models.py]: These classes are way more concise, nice work. ^

File: project/static/wibi/README.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # Wibi
 
-This README outlines the details of collaborating on this Ember application.
-A short introduction of this app could easily go here.
+This outlines the frontend Angular JS WIBI application
```
Comment (Borgaard) [inline project/static/wibi/README.md]: Short introduction noted.

**Comment by natsn** [comment]
Thanks @Borgaard !

**Pull request merged**